build/tools/Automation/Python/Constants.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace Oryx.Microsoft.Automation.Python
+{
+    public class Constants
+    {
+        public const string ConstantsYaml = "constants.yaml";
+        public const string PythonName = "python";
+        public const string VersionsToBuildTxt = "versionsToBuild.txt";
+        public const string YamlPythonKey = "python-versions";
+        public static readonly Dictionary<string, string> VersionGpgKeys = new Dictionary<string, string>()
+        {
+            { "3.10", "A035C8C19219BA821ECEA86B64E628F8D684696D" },
+            { "3.11", "A035C8C19219BA821ECEA86B64E628F8D684696D" },
```

Comment:
Are these values that we'll need to update for every new release of Python 3.x or are they placeholders?

Reply:
Every couple minor versions we'll have to update these. For example, 3.10.x and 3.11.x have the same key.

This observation is from here:
https://www.python.org/downloads/
![image](https://user-images.githubusercontent.com/86327553/221330240-44d4a456-fb96-4bbf-9da9-12e96578d565.png)

Are you thinking we should store these differently?